packages/lab/src/calendar/internal/CalendarDay.css

```diff
@@ -29,7 +29,13 @@
 .saltCalendarDay-outOfRange {
   color: var(--salt-content-secondary-foreground);
 }
+.saltCalendarDay-outOfRange .saltCalendarDay-today {
+  border-bottom: var(--salt-size-indicator) var(--salt-selectable-borderStyle) var(--salt-content-secondary-foreground);
+}
 
+.saltCalendarDay-outOfRange .saltCalendarDay-highlighted {
```

Comment:
I might be mistaken but this doesn't work? the outOfRange class and highlighted class are on the same element.